### DISCUSSION TITLE
Please How i Change ip for all profile i have?

### DISCUSSION THREAD

**Discussion by mohammedxhameed** [pivpn/pivpn#1537] [Support]
Hello, I want a way to change the IPs after installation

Naturally, the IP is generated randomly after installation. I want a way to change the IPs without cutting the Internet to the customers, because I tried over and over again, the Internet will be cut off for them, but the IPs change

**Comment by coolapso** [comment]
Please be more specific, 

What IPs? 
The ip's the clients connect to (The public ip)?? 
The internal IP? 

you should have read this: https://github.com/pivpn/pivpn#readme
And this: https://docs.pivpn.io before opening the issue. 

Moving to discussions.

**Reply by mohammedxhameed** [reply]
when install pivpn the privet ip is 10.0.54.0/24

how i change this ips to all clients? like to 10.8.0.0/24

**Comment by mohammedxhameed** [comment]
when install pivpn the privet ip is 10.0.54.0/24

how i change this ips to all clients? like to 10.8.0.0/24

**Reply by orazioedoardo** [reply]
OpenVPN or WireGuard? Anyway I don't think you can change the subnet without the clients restarting their connection.
